### ISSUE TITLE
Use More Interesting Datasets in Tutorial Notebooks

### ISSUE THREAD

**Issue by wilsonbb** [lincc-frameworks/tape#291]
In the tutorial notebooks, we're often using some rather uninteresting datasets to explain TAPE functionality (see how [Working with the Ensemble](https://tape.readthedocs.io/en/latest/tutorials/working_with_the_ensemble.html) uses from_source dict and produces an object table with no columns other than its index). We should be able to provide some small but more compelling datasets.

**Comment by hombit** [comment]
We could load some data from [Vizier](http://vizier.nao.ac.jp/viz-bin/VizieR) via [`astroquery`](https://astroquery.readthedocs.io/en/latest/vizier/vizier.html) or from ZTF querying [IPAC](https://irsa.ipac.caltech.edu/docs/program_interface/ztf_lightcurve_api.html) (for DR) or using [Alerce](https://pypi.org/project/alerce/)/[ANTARES](https://pypi.org/project/antares-client/)/[Fink](https://pypi.org/project/fink-client/) client (for alerts).

For Vizier we could use one of SDSS Stripe 82 datasets familiar for us or any of many other datasets.

**Comment by dougbrn** [comment]
This is partially the motivation for our datasets stored in the ill-named https://github.com/lincc-frameworks/tape_benchmarking repo. We could use one of the existing ones here, with the caveat that they handle the column mapping for you and that makes them less representative of common user workflows, or add another dataset to it?

**Comment by hombit** [comment]
@dougbrn I agree about the `tape_benchmakrking` repo, but I believe it would nice to load "raw" well-known dataset from the web for the tutorial.

**Comment by dougbrn** [comment]
Fair point!
